pynecone/utils.py

```diff
@@ -503,9 +507,40 @@ def export_app(app: App, zip: bool = False):
     # Zip up the app.
     if zip:
         if os.name == "posix":
-            cmd = r"cd .web/_static && zip -r ../../frontend.zip ./* && cd ../.. && zip -r backend.zip ./* -x .web/\* ./assets\* ./frontend.zip\* ./backend.zip\*"
+            posix_export(backend, frontend)
         if os.name == "nt":
-            cmd = r'''powershell -Command "Set-Location .web/_static; Compress-Archive -Path .\* -DestinationPath ..\..\frontend.zip -Force;cd ..\..;Get-ChildItem .\* -Directory  | where {`$_.Name -notin @('.web', 'assets', 'frontend.zip', 'backend.zip')} | Compress-Archive -DestinationPath backend.zip -Update"'''
+            nt_export(backend, frontend)
+
+
+def nt_export(backend, frontend):
```

Comment:
Can we add type annotations to the args here

Reply:
done